README.md

```diff
@@ -68,7 +68,7 @@ cargo build --release
 
```

Comment:
Can you update the README instructions with the new commands to get the UI demo up and running?
i.e.
AS a regular node vs as a mining node. Or is the demo not as easy to deploy with a pow chain?


From your cel implementation: 
`As a mining node: ./cle-coin --chain=mainnet-spec.json --name YOUR-NODE-NAME --validator`

Reply:
UI demo is now working again. :+1: